client/js/render.js

```diff
@@ -35,8 +36,10 @@ function buildChatMessage(data) {
 		target = "#chan-" + chat.find(".active").data("id");
 	}
 
-	if (data.msg.preview) {
-		data.msg.preview.shown = options.shouldOpenMessagePreview(data.msg.preview.type);
+	if (data.msg.previews.length) {
+		data.msg.previews.forEach((preview) => {
+			preview.shown = options.shouldOpenMessagePreview(preview.type);
```

Comment:
What do you mean? I haven't improved/changed the UI one bit, so before that PR, if you were posting a URL and that would show as open, with this PR that same behavior applies for multiple URLs.
One reason that I didn't touch UI is because you are improving it in #1301 and I think we should keep those improvements there.

I'm still debating whether we want the caret right next to the URLs (à la Slack) or not, but to be honest the current UI works fairly well even with multiple URLs. Sure, it's taking a lot of space for nothing, but I'm not saying it's definitive :)